templates/al2023/runtime/gpu/nvidia-kmod-load.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Loading Nvidia kernel modules
+After=nodeadm-run.service
```

Comment:
Can you drop a blurb here about what we're waiting for nodeadm to do before starting?

Reply:
Hm, thinking about it again, we don't really need to wait for nodeadm to setup the node. Loading the modules and nodeadm doing its thing can be done at the same time right?